check_tv.py

```
@@ -522,26 +522,24 @@ def create_live_stream(title, description, kmself):  # Function to create live s
                     )
                     playlist_request.execute()  # Execute playlist request
-                    logging.info(f"Successfully added video {video_id} to playlist {config.playlist_id1}")  # Log success
+                    logger.info(f"Successfully added video {video_id} to playlist {config.playlist_id1}")  # Log success
                 except Exception as playlist_error:  # Handle exceptions
-                    logging.error(f"Failed to add video to playlist: {playlist_error}")  # Log error
+                    logger.error(f"Failed to add video to playlist: {playlist_error}")  # Log error
             return video_id  # Return video ID
         except Exception as e:  # Handle exceptions
           if hitryagain == 3:  # Check if retry limit is reached
-           logging.info(f"Error and stoping because of error that can't fix")  # Log error
+           logger.info(f"Error and stoping because of error that can't fix")  # Log error
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)